modules/workers/instanceconfig.tf

```diff
@@ -6,21 +6,17 @@ resource "oci_core_instance_configuration" "workers" {
   for_each       = local.enabled_instance_configs
   compartment_id = each.value.compartment_id
   display_name   = each.key
+  defined_tags   = each.value.defined_tags
+  freeform_tags  = each.value.freeform_tags
 
   instance_details {
     instance_type = "compute"
 
-
-
     launch_details {
```

Comment:
I don't see agent configuration in the instance config

Reply:
This hasn't been implemented/tested yet. It'd be a good addition, but let's do it in a separate change.